app/jobs/govbox/sync_folder_job.rb

```diff
@@ -15,6 +15,8 @@ def perform(folder, upvs_client: UpvsEnvironment.upvs_client, batch_size: 1000)
         moved_edesk_message_ids = []
 
         raw_messages.each do |raw_message|
+          next if folder.box.settings['sync_since'].present? && (Date.parse(raw_message['delivered_at']) < Date.parse(folder.box.settings['sync_since']))
```

Comment:
Toto parsovanie sync_since vytiahnime mimo each nech sa to parsuje len raz.

Reply:
Upravene